sencha-workspace/SlateDemonstrationsTeacher/app/controller/Dashboard.js

```diff
@@ -47,10 +47,41 @@ Ext.define('SlateDemonstrationsTeacher.controller.Dashboard', {
     ],
 
     refs: {
-        dashboardCt: 'slate-demonstrations-teacher-dashboard',
-
         studentProgressGrid: 'slate-demonstrations-teacher-dashboard slate-demonstrations-teacher-studentsprogressgrid',
-        teacherOverviewwindow: 'slate-demonstrations-teacher-skill-overviewwindow'
+        teacherOverviewwindow: 'slate-demonstrations-teacher-skill-overviewwindow',
+        dashboardCt: {
+            selector: 'slate-demonstrations-teacher-dashboard',
+            autoCreate: true,
+
+            xtype: 'slate-demonstrations-teacher-dashboard'
+        }
```

Comment:
@aliyeysides refs should be grouped where there is hierarchy. As you can see in the selectors, the `studentsProgressGrid` is a subcomponent of `dashboardCt` that gets created by it so we should group/order them as seen in the commit i'll make